lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/classify/MultiMADClassifier.java

```diff
@@ -0,0 +1,292 @@
+package edu.stanford.futuredata.macrobase.analysis.classify;
+
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+import edu.stanford.futuredata.macrobase.operator.Transformer;
+
+import org.apache.commons.math3.special.Erf;
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+
+import java.lang.Math;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class MultiMADClassifier implements Transformer {
+    private double percentile = 0.5;
+    private double zscore = 2.576;
+    private boolean includeHigh = true;
+    private boolean includeLow = true;
+    private double samplingRate = 1;
+    private List<String> columnNames;
+    private String outputColumnSuffix = "_OUTLIER";
+    private boolean useParallel = false;
+    private boolean useReservoirSampling = false;
+    private final double trimmedMeanFallback = 0.05;
+    // https://en.wikipedia.org/wiki/Median_absolute_deviation#Relation_to_standard_deviation
+    private final double MAD_TO_ZSCORE_COEFFICIENT = 1.4826;
+
+    // Calculated values
+    private double lowCutoff;
```

Comment:
Since there is no getter for lowCutoff and highCutoff, maybe remove them from class variables?